src/test/java/emissary/core/IBaseDataObjectDiffHelperTest.java

```diff
@@ -294,4 +298,20 @@ void testDiffList() {
         ibdo2.setClassification("classification");
         verifyDiffList(1, ibdoList1, ibdoList2);
     }
+
+    @Test
+    void testParamSort() {
+        // set-up
+        Set<String> expectedParams = new HashSet<>();
```

Comment:
Should probably use a `TreeSet` for the same reason we're now using a `TreeMap`
```suggestion
        Set<String> expectedParams = new TreeSet<>();
```

Reply:
Dang you right, let me do it real quick